source/uri-options/tests/srv-options.yml

```diff
@@ -24,39 +24,39 @@ tests:
           srvMaxHosts: 2
     - description: "SRV URI with negative integer for srvMaxHosts"
       uri: "mongodb+srv://test1.test.build.10gen.cc/?srvMaxHosts=-1"
-      valid: false
+      valid: true
```

Comment:
I was expecting:

```
valid: false
warning: false
```

Reply:
This seems consistent with some other tests for invalid values, such as [`tlsAllowInvalidCertificates`](https://github.com/mongodb/specifications/blob/master/source/uri-options/tests/tls-options.yml#L40).